app/controllers/hub/organizations_controller.rb

```diff
@@ -0,0 +1,48 @@
+module Hub
+  class OrganizationsController < ApplicationController
+    include AccessControllable
+    before_action :require_sign_in
+    load_and_authorize_resource :vita_partner, parent: false
+
+    layout "admin"
+
+    def new
+      @coalitions = Coalition.all
+    end
+
+    def create
+      @coalitions = Coalition.all
+      if @vita_partner.save
+        redirect_to hub_organizations_path
+      else
+        render :new
+      end
+    end
+
+    def index
+      @coalitions = Coalition.includes(:organizations)
+      @organizations = @vita_partners.organizations
+      @independent_organizations = @vita_partners.organizations.where(coalition: nil)
+    end
+
+    def edit
+      @coalitions = Coalition.all
+      @organization = @vita_partner
+    end
+
+    def update
+      render :edit unless @vita_partner.update(vita_partner_params)
+
+      redirect_to hub_organizations_path
+    end
+
+    private
+
+    def vita_partner_params
+      params.require(:vita_partner).permit(:name, :coalition_id).merge(
+        zendesk_group_id: "unused",
```

Comment:
gross -- hoping we can delete that dead code soon!

Reply:
I have a local commit ready for that :D as soon as this gets merged :D